test/integration/image-optimizer/test/index.test.js

```diff
@@ -15,9 +16,9 @@ import {
 import isAnimated from 'next/dist/compiled/is-animated'
 import { join } from 'path'
 
-jest.setTimeout(1000 * 60 * 2)
+jest.setTimeout(1000 * 60 * 5)
 
-const appDir = join(__dirname, '../')
+const appDir = join(__dirname, '../app')
```

Comment:
Why was this moved to `app`? Typically the test fixture is the app.

Reply:
We do this in some of the test fixtures to help separate the test app from the test code like [here](https://github.com/vercel/next.js/tree/canary/test/integration/with-electron/app) since installing certain `node_modules` at the same level as the `test` fixture folder can cause issues with jest